frontend/src/components/pages/AddProduct.js

```diff
@@ -0,0 +1,232 @@
+                        <input
+                            type="text"
+                            id="title"
+                            required
+                            value={product.title}
+                            onChange={handleInputChange}
+                            name="title"
+                            placeholder="Coqueteleira"
+                        />
+                    </div>
+                
+                    <div className="form-group">
+                        <label htmlFor="description">Descrição</label> <br/>
+                        <input
+                            type="text"
+                            id="description"
+                            required
+                            value={product.description}
+                            onChange={handleInputChange}
+                            name="description"
+                            placeholder="Ex: Coqueteleira 500ml de aço inox"
+
+                        />
+                    </div>
+
+                    <div className="form-group">
+                        <label htmlFor="price">Preço</label> <br/>
+                        <input
+                            type="text"
+                            id="price"
+                            required
+                            value={product.price}
+                            onChange={handleInputChange}
+                            name="price"
+                            placeholder="Ex: 99,99"
+                        />
+                    </div>
+
+                    <div className="form-group">
+                        <label htmlFor="stock">Quantidade em Estoque</label> <br/>
+                        <input
+                            type="text"
+                            id="stock"
+                            required
+                            value={product.stock}
+                            onChange={handleInputChange}
+                            name="stock"
+                            placeholder="Ex: 50"
+                        />
+                    </div>
+
+                    <div className="form-group">
+                        <label htmlFor="category">Categoria</label> <br/>
+                        <input
+                            type="text"
+                            id="category"
+                            required
+                            value={product.categoryId}
+                            onChange={handleInputChange}
+                            name="categoryId"
+                            placeholder="Ex: 2"
+                        />
+                    </div>
+
+                    <br/> <button onClick={saveProduct} className="btn btn-sucess">
+                        Cadastrar Produto    
+                    </button>
+                    <br/> <button onClick={saveProduct} className="btn btn-cancel">
+                        Cancelar    
+                    </button>                            
```

Comment:
Aqui tem algumas coisas para comentar:

1. Notei um typo na palavra "_sucess_", acredito que deveria ser "success":

```suggestion
                    <br/> <button onClick={saveProduct} className="btn btn-sucess">
                        Cadastrar Produto    
                    </button>
                    <br/> <button onClick={saveProduct} className="btn btn-cancel">
                        Cancelar    
                    </button>                            
```

Reply:
2. Você está usando chamando a função `saveProduct` no click do botão "Cancelar", isso não parece ser certo.

Reply:
3. Se você usar o `form` por volta desse botões e inputs, você pode facilitar esse código também:

```suggestion
                    <br/> <button type"submit" className="btn btn-success">
                        Cadastrar Produto    
                    </button>
                    <br/> <button type="reset" className="btn btn-cancel">
                        Cancelar    
                    </button>                            
```

O `type="submit"` ira submeter o formulário, então vai chamar a função que você passar no `onSubmit` do form.
O `type="reset"` ira limpar todos os campos do formulário.

Reply:
> 
Eu esqueci de remover essa função kkkk eu coloquei só pra deixar o botão funcional. Vou resolver ela num outro PR junto da alteração pro form